addons/web/static/src/legacy/js/assets.js

```diff
@@ -0,0 +1,23 @@
+odoo.define('web.assets.qweb', function (require) {
+"use strict";
+
+const {assets} = require('@web/core/assets');
+
+const loadXML = assets.loadXML;
+assets.loadXML = function (xml, app=assets.defaultApp) {
```

Comment:
Can you remind me why you need to patch this in place?

Reply:
All bundles go through the load method which is exported. By modifying/patching the method on assets, we are sure that the behavior is modified for all calls. In order for qweb templates to be loaded as well, the behavior of this method must be modified. It's in the legacy, so it can just be removed when qweb goes away.